disasm/disasm.go

```diff
@@ -0,0 +1,324 @@
+// Copyright 2017 The go-interpreter Authors.  All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// package disasm provides functions for disassembling WebAssembly bytecode.
```

Comment:
Minor nitpick, but `golint` would complain about this:

```
package comment should be of the form "Package disasm ..."
```

The first word of the comment sentence should begin with a capital letter.